doc/hardware/porting/board_porting.rst

```diff
@@ -8,15 +8,69 @@ directory* with various files in it. Files in the board directory inherit
 support for at least one SoC and all of its features. Therefore, Zephyr must
 support your :term:`SoC` as well.
 
+.. _board_and_identifiers
+
+Board and board identifiers
+***************************
+
+A board may be a physical piece of hardware or an emulated board.
+Furthermore a board may contain just a single SoC or it may contain multiple
+SoCs, as well as each SoC may be contain a single or multiple CPU cores.
+
+A board may further define multiple variants for various use-cases.
```

Comment:
Do I understand correctly that the "variant" is a property of the "soc" property of the "board"?
(This paragraph, coming first, would seem to indicate there is also a "variant" property of the "board" itself)

Reply:
no, the variant is applied after the SoC (or cpucluster when such is defined).

How about shortening to: `It's possible to define variants for dedicated use-cases.` ?

Reply:
suggestion applied